equationsarea.cpp

```diff
@@ -1,6 +1,135 @@
 #include "equationsarea.h"
 
-EquationsArea::EquationsArea(QWidget *parent) : QWidget(parent)
+EquationsArea::EquationsArea(QWidget *parent) : QWidget(parent),
+    background_color(255,255,255),
+    graph_color(0,0,0),
+    Graph(Line)
 {
+      load_graph_change();
+}
+
+QPointF EquationsArea::drawGeneral(float t)
+{
+    ///break statements have been removed since each case returns a QPointF function
```

Comment:
consistency:

break -> Break
function -> function.

Small thing that we all can touch up at the end.